react-components/intro-to-react-js.md

```diff
@@ -0,0 +1,101 @@
+# Intro to React JS
+
+## Learning Goals
+
+- What is React?
+- Why do we learn React?
+- Who is using React?
+- Intro to the Virtual DOM
+- Intro to Components
+
+## Introduction
+
+Consider a social media website, such as [Reddit.com](https://www.reddit.com/r/reactjs/). Every given web page needs to do soooo many things! To name a few, each web page needs to:
```

Comment:
restrict pages under discussion to those on Reddit.
"things" doesn't feel like a concrete-enough noun to omit it from the "To name a few" phrase, but adding it in sounds redundant. I avoided the issue by switching over to "for example".
```suggestion
Consider a social media website, such as [Reddit.com](https://www.reddit.com/r/reactjs/). Every web page on that site needs to do soooo many things! For example, each web page needs to:
```